packages/lodestar/src/network/subnets/syncnetsService.ts

```diff
@@ -82,7 +83,9 @@ export class SyncnetsService implements ISubnetsService {
     if (prevFork === ForkName.phase0) return;
     this.logger.info("Unsuscribing to random attnets from prev fork", {prevFork});
     for (let subnet = 0; subnet < SYNC_COMMITTEE_SUBNET_COUNT; subnet++) {
-      this.gossip.unsubscribeTopic({type: gossipType, fork: prevFork, subnet});
+      if (!this.opts?.subscribeAllSubnets) {
```

Comment:
Maybe we should have a short circuit at the top of this and `unsubscribeSubnets`
`if (this.opts?.subscribeAllSubnets) return;`

Reply:
I think this syntax is more obvious, since it's immediately closer to the unsubscribeTopic() topic call.